main.js

```diff
@@ -0,0 +1,35 @@
+var swiper = new Swiper('.swiper--preview-img', {
+  loop: true,
+  spaceBetween: 10,
+  slidesPerView: 4,
+  freeMode: true,
+  watchSlidesProgress: true,
+
+});
+
+var swiper = new Swiper('.swiper--view-img', {
```

Comment:
No esta bien declarar la misma variable dos veces
```suggestion
const swiperView = new Swiper('.js-swiper-view-img', {
```